docs/learn/digital-assets/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: '👋🏻 Getting Started'
+sidebar_position: 1
+description: Create a Hardhat project and start building your smart contracts for LUKSO.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started building contracts
+
+The LUKSO ecosystem offers smart contract developers a lot of [new standards](../../standards/introduction.md) and [tools](../../tools/getting-started.md) to build **powerful, modular, and standardized** blockchain applications.
+
+This page will guide you through setting up your development environment to build on LUKSO.
+
+If you need more low level information about our contracts, you can check the dedicated [contracts](../../contracts/introduction.md) section.
+
+### Playground Repository
+
+In the following guides, we will use the [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository with a full Hardhat setup, including ready-to-go network configurations, sample contracts, and scripts to deploy and verify LSP-based contracts on LUKSO.
+
+<div style={{textAlign: 'center'}}>
+
+<img
+src="/img/guides/playground_hardhat.png"
+alt="LUKSO Playground Hardhat"
+/>
+
+</div>
+
+## Clone the Playground Repository
+
+First, clone the playground repository and navigate into the smart-contracts-hardhat folder:
+
+```bash
+git clone https://github.com/lukso-network/lukso-playground.git
+cd lukso-playground/smart-contracts-hardhat
+```
+
+Once navigated, you have a working Hardhat setup.
+
+:::info
+
+**This repository uses the [bun](https://bun.sh) toolkit to manage packages and run scripts. Make sure to [install it](https://bun.sh/docs/installation) first.**
+
+:::
+
+To install all the dependencies, run the below command:
+
+```bash
+bun install
+```
+
+You can then go ahead to create a `.env` file within the root folder of the Hardhat repository and the following contents:
+
+```text title=".env"
+PRIVATE_KEY=0x...
+UP_ADDR=0x...
+```
+
+The private key will be used to deploy the smart contracts to the network and can either be a **standalone Externally Owned Account** or a **controller of a Universal Profile**. If you want to deploy the smart contracts as a Universal Profile, you also have to provide the related address of the Universal Profile.
```

Comment:
We are not showing guides to deploy as a UP. Only as an EOA. Developers looking to know how to deploy a contract using a UP can look at this guide:

https://docs.lukso.tech/learn/universal-profile/interactions/deploy-contracts

We can add the code snippet to deploy a token from a UP as a sub-category "Example" in this guide